yt_geotiff/tsearch.h

```diff
@@ -0,0 +1,26 @@
+/*

```

Comment:
Just to be clear, this file serves no other purpose than I get a compile error if it is not here.